### PR TITLE
Added functionality for the spanish deck

### PR DESCRIPTION
**Pull request by gilesni**

### CODE REVIEW

**Comment by Brenisec** [comment]
well done. looks like the code needs to be tested with our tests. But the test should run fine as the code is coherent.

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -47,6 +47,10 @@ public void remove(int columnNumber) {
                                 removeCard = true;
                             }
                         }
+                        else if (compare.getSuit() == Suit.Comodines){
```
Comment (Brenisec) [inline src/main/java/models/Game.java]: good check for suit type. very clean

File: src/main/java/models/Sdeck.java [hunk 2]
```diff
@@ -8,15 +8,26 @@
     public java.util.List<Card> cards = new ArrayList<>();
 
     public Sdeck() {
-        for(int i = 2; i < 14; i++){
-            cards.add(new Card(i,Suit.Bastos));
-            cards.add(new Card(i,Suit.Oros));
-            cards.add(new Card(i,Suit.Copas));
-            cards.add(new Card(i,Suit.Espadas));
+        for (int i = 2; i < 14; i++) {
```
Comment (Brenisec) [inline src/main/java/models/Sdeck.java]: very straight forward and easy to read.

File: src/main/java/models/Deck.java [hunk 3]
```diff
@@ -22,14 +32,6 @@ public void shuffle() {
...
-    public ArrayList<Card> dealFour() {
-        ArrayList<Card> four = new ArrayList<>();
-        for (int i = 0; i < 4; i++) {
-            if (cards.size() > 0) {
-                four.add(cards.get(i));
-                cards.remove(i);
-            }
-        }
-        return four;
+
```
Comment (hutchico) [inline src/main/java/models/Deck.java]: Wait, so we took the deal code and it somewhere else?

**Comment by hutchico** [comment]
From a code standpoint, this looks good. The game class still needs a way to swap between the two deck types, I'll get that.

**Pull request merged**